src/Tokenizers/PHP.php

```diff
@@ -589,6 +589,58 @@ protected function tokenize($string)
                 echo PHP_EOL;
             }
 
+            if ($tokenIsArray === true
+                && isset(Util\Tokens::$contextSensitiveKeywords[$token[0]]) === true
+                && isset($this->tstringContexts[$finalTokens[$lastNotEmptyToken]['code']]) === true
+            ) {
+                $preserveKeyword = false;
+
+                // `new class` should be preserved
+                if ($token[0] === T_CLASS && $finalTokens[$lastNotEmptyToken]['code'] === T_NEW) {
+                    $preserveKeyword = true;
+                }
+
+                // `new class implements` should be preserved
+                if ($finalTokens[$lastNotEmptyToken]['code'] === T_CLASS) {
+                    $preserveKeyword = true;
+                }
+
+                // `namespace\` should be preserved
+                if ($token[0] === T_NAMESPACE) {
+                    for ($i = ($stackPtr + 1); $i < $numTokens; $i++) {
+                        if (is_array($tokens[$i]) === false) {
+                            break;
+                        }
+
+                        if (isset(Util\Tokens::$emptyTokens[$tokens[$i][0]]) === true) {
+                            continue;
+                        }
+
+                        if ($tokens[$i][0] === T_NS_SEPARATOR) {
+                            $preserveKeyword = true;
+                        }
+
+                        break;
+                    }
+                }
+
+                if ($preserveKeyword === false) {
+                    if (PHP_CODESNIFFER_VERBOSITY > 1) {
+                        $type = $tokens[0];
```

Comment:
This should be `$token[0]` (not the `s`). This causes an error with verbosity on.

Reply:
Actually, this would be better as:
```
$type = Util\Tokens::tokenName($token[0]);
```
So that the token name shows up like in other verbose messages. It just shows an INT at the moment.

Reply:
Fixed.